### PR TITLE
add metrics-server v0.6.1

### PR DESCRIPTION
**Pull request by ggpaue**
## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

**Review by seemiller — CHANGES_REQUESTED** [review]
I think the `test.yaml` file is extraneous, and it is causing an error with the overlays.
```
cd addons/packages/metrics-server/0.6.1/bundle/config
ytt -f .
```
yields the error:
```
ytt: Error: Overlaying (in following order: overlays/overlay-apiservice.yaml, overlays/overlay-deployment.yaml, overlays/overlay-namespace.yaml, overlays/update-strategy-overlay.yaml):
  Document on line overlays/overlay-apiservice.yaml:5:
    Expected number of matched nodes to be 1, but was 2 (lines: test.yaml:184, upstream/components.yaml:181)
```

File: addons/packages/metrics-server/0.6.1/package.yaml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  refName: metrics-server.community.tanzu.vmware.com
+  version: 0.6.1
+  releaseNotes: metrics-server 0.6.1 https://github.com/kubernetes-sigs/metrics-server/releases/tag/v0.6.1
+  licenses:
+  - Apache 2.0
+  template:
+    spec:
+      fetch:
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/metrics-server@sha256:4fa580c9b185d44f39847cd2347c8110b3a38d775e674ce77bc3b2569067f4c0
```
Comment (ggpaue) [inline addons/packages/metrics-server/0.6.1/package.yaml:133]: @seemiller should this image be manually pushed?

**Comment by ggpaue** [comment]
> I think the `test.yaml` file is extraneous, and it is causing an error with the overlays.
> 
> ```
> cd addons/packages/metrics-server/0.6.1/bundle/config
> ytt -f .
> ```
> 
> yields the error:
> 
> ```
> ytt: Error: Overlaying (in following order: overlays/overlay-apiservice.yaml, overlays/overlay-deployment.yaml, overlays/overlay-namespace.yaml, overlays/update-strategy-overlay.yaml):
>   Document on line overlays/overlay-apiservice.yaml:5:
>     Expected number of matched nodes to be 1, but was 2 (lines: test.yaml:184, upstream/components.yaml:181)
> ```

ah I added by mistake, now deleted it

**Review by seemiller — CHANGES_REQUESTED** [review]

File: addons/packages/metrics-server/0.6.1/package.yaml (hunk 1 above)
Comment (ggpaue) [reply addons/packages/metrics-server/0.6.1/package.yaml]: thank you very much! @seemiller, I have pushed the changed SHA

**Review by seemiller — APPROVED** [review]

**Pull request merged**